### ISSUE TITLE
Update comments part failing

### ISSUE THREAD

**Issue by AhirGhosh** [begyyal/act_revise_comments#10]
Hi, while prefixing the comment, it's failing in here:
`Run begyyal/act_revise_comments@v1.0.0
  with:
    prefix: Hello
    userName: Ahir
    token: ***
    repos: AhirGhosh/github-actions
    userMail: begyyal@gmail.com
/usr/bin/docker run --name bcf0995b57eb60ed1485cb3a3bef955c84ea1_ba5698 --label [2](https://github.com/AhirGhosh/github-actions/runs/5796311981?check_suite_focus=true#step:3:2)bcf09 --workdir /github/workspace --rm -e INPUT_PREFIX -e INPUT_USERNAME -e INPUT_BRANCH -e INPUT_TOKEN -e INPUT_REPOS -e INPUT_FROM -e INPUT_TO -e INPUT_USERMAIL -e INPUT_OMITSPACEFLAG -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/github-actions/github-actions":"/github/workspace" 2bcf09:95b57eb60ed1485cb[3](https://github.com/AhirGhosh/github-actions/runs/5796311981?check_suite_focus=true#step:3:3)a3bef955c8[4](https://github.com/AhirGhosh/github-actions/runs/5796311981?check_suite_focus=true#step:3:4)ea1  "Hello" "" "***" "AhirGhosh/github-actions" "" "" "Ahir" "begyyal@gmail.com" ""
Cloning into 'github-actions'...
fatal: could not read Username for 'https://github.com/': No such device or address
/entrypoint.sh: line 33: cd: ./github-actions: No such file or directory
fatal: --local can only be used inside a git repository
fatal: --local can only be used inside a git repository
fatal: --local can only be used inside a git repository
fatal: not a git repository (or any parent up to mount point /github)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: not a git repository (or any parent up to mount point /github)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: not a git repository (or any parent up to mount point /github)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Unique commit of the [to] branch from [from] branch as the target of revision dont exist.`

**Comment by AhirGhosh** [comment]
My workflow yml is as below:
`name: Append Comments

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

  workflow_dispatch:

jobs:
  update-comments:
    runs-on: ubuntu-latest
    steps:
      - name: Give a prefix to comments
        uses: begyyal/act_revise_comments@v1.0.0
        with:
         prefix: Hello
         userName: Ahir`

**Comment by AhirGhosh** [comment]
![image](https://user-images.githubusercontent.com/40377866/161356979-08be9996-dbba-4f04-ae8e-484602497b8b.png)

**Comment by begyyal** [comment]
Thanks for your comments.
That seems a bug to failure cloning private repository.
So I reproduce the error when use private repository.

It's fixed, please try to run at `v1.0.1` or `v1`.

**Comment by AhirGhosh** [comment]
Thanks @begyyal for the quick response and fixing it so fast, you are genius. My same workflow as above succeeded once I used v1.
Just one question regarding the usage, I think this appends the prefix to all commits' comments in every branch?
So if my commit comment is "Testing", then this workflow will update the commit comment as "<Prefix> Testing.

Please let me know if my understanding is correct.

**Comment by begyyal** [comment]
It's depends on the conditions, i'm sorry that the description is difficult to understand.

At first, **this action was created on the assumption that it targets the differences between two branches**.
So mainly had suppose to pull_request trigger as use case.

In push trigger case, it's no appending if use your workflow file.
It needs to set params [branch/from](https://github.com/begyyal/act_revise_comments/blob/master/action.yml) in order to appned a prefix.
Because these params default values are [github.head_ref/github.base_ref](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) that can be used on pull_request only.

But I realised that it might be better to follow simple push trigger case, it's perhaps possible by using `github.event.commits`.

As a result, I correspond to belows in order.
1. modify the desctiption(readme) to add above things.
2. consider to update implementation to follow simple push trigger case, with version update to v2 if interface changed.

Thanks to commnets, I was able to gain awareness.
